carbonmark/components/Layout/AddressSection/index.tsx

```diff
@@ -21,13 +25,19 @@ export const AddressSection: FC<AddressSectionProps> = (props) => {
           <Trans>Not Connected</Trans>
         </Text>
       )}
-      {props.address && (
-        <CopyAddressButton
-          label={concatAddress(props.address)}
-          address={props.address}
-          className="copyButton"
-        />
-      )}
+      {props.address &&
+        (isTorus ? (
+          <ViewWalletButton
+            address={props.address}
+            href="https://polygon.tor.us/"
```

Comment:
Nit: Can remove the hardcoded href and use the url which already exists in the constants file.

```
import { urls } from "@klimadao/lib/constants";

<ViewWalletButton
  address={props.address}
  href={urls.polygonTor}
/>
```

Reply:
good point! thanks